pkg/core/resources/apis/meshexternalservice/api/v1alpha1/meshexternalservice.go

```diff
@@ -94,8 +98,9 @@ const (
 
 type Verification struct {
 	// Mode defines if proxy should skip verification, one of `SkipSAN`, `SkipCA`, `Secured`, `SkipAll`. Default `Secured`.
+	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=Secured
-	Mode *VerificationMode `json:"mode,omitempty"`
+	Mode VerificationMode `json:"mode"`
```

Comment:
maybe it is worth adding an enum annotation here

Reply:
If we add enum then universal zones that did not have that set can send empty mode and that would mean error on kubebuilder validation right? soooo ideally we would have to write a migration for that

Reply:
we agreed on slack to not add enums yet, will create an issue for this